dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5789,6 +5789,21 @@ public String toString() {
           .setIsHidden(true)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .build();
+  public static final PropertyKey USER_POSITION_READER_STREAMING_ADAPTIVE_POLICY_ENABLED =
+      booleanBuilder(Name.USER_POSITION_READER_STREAMING_ADAPTIVE_POLICY_ENABLED)
+          .setScope(Scope.CLIENT)
+          .setDefaultValue(false)
+          .setDescription("If uses adaptive policy to adjust the prefetch window size")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
```

Comment:
I think this one is also experimental. When we settle down on a relatively stable interfaces for the prefetch policies, we can use a enum type that allows a user to specify a particular kind of policy.

```suggestion
          .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
          .setIsHidden(true)
```